minho/programmers/Level_1/3일차/K번째_수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, commands) {  
+    let ar = [];
+    for(let com_el of commands){
+        let el = array.slice(com_el[0] - 1,com_el[1]).sort(function(a, b){return a - b});        
+        el = el[com_el[2] - 1];
+        ar.push(el);        
+    }        
```

Comment:
```suggestion
    for (let com_el of commands){
        let el = array.slice(com_el[0] - 1, com_el[1]).sort((a, b) => {
        return a - b;
        });        
        el = el[com_el[2] - 1];
        ar.push(el);        
    }        
```
for문 컨벤션 지켜주세요!
function을 써서 해주는것도 좋지만 제가 금요일에 알려드린 앨리쌤 arrow function보고 하시면 더 편하게 코드 작업 하실 수 있으실거에요ㅎㅎ